BrandonBeltran/hackerrank/03/RandomNoteSolve.java

```diff
@@ -0,0 +1,41 @@
+package bootcamp;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RandomNoteSolve {
+
+    public static void checkMagazine(List<String> magazine, List<String> note) {
+
+        Map<String, Integer> magazineWords = wordsToHashMap(magazine);
```

Comment:
Best practice in Collections or Map Variable Name, is to use singular name and interface name: numberList, change it in all variables